src/coreclr/src/inc/sospriv.idl

```diff
@@ -390,3 +390,19 @@ interface ISOSDacInterface7 : IUnknown
     HRESULT GetProfilerModifiedILInformation(CLRDATA_ADDRESS methodDesc, struct DacpProfilerILData *pILData);
     HRESULT GetMethodsWithProfilerModifiedIL(CLRDATA_ADDRESS mod, CLRDATA_ADDRESS *methodDescs, int cMethodDescs, int *pcMethodDescs);
 };
+
+[
+    object,
+    local,
+    uuid(c12f35a9-e55c-4520-a894-b3dc5165dfce)
+]
+interface ISOSDacInterface8 : IUnknown
```

Comment:
I would suggest also adding an `GetNumberGenerations` API since this number is now variable based on the runtime.  It would be odd to have to call `GetGenerationTable` as a way to find out the number of generations.  Having it be its own API is much cleaner, and would be used by ClrMD.

Reply:
Sure, no problem